database/schema.sql

```diff
@@ -11,7 +11,7 @@ CREATE TABLE circuit(
 );
 
 CREATE TABLE judge(
-    "judge_id" SMALLINT UNIQUE PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    "judge_id" INT UNIQUE PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
```

Comment:
Im guessing you tried to keep it consistent with changing it to INT, incase theres long ids, or titles.